conf/functions_worker.yml

```diff
@@ -27,7 +27,7 @@ workerPort: 6750
 workerPortTls: 6751
 
 # Configuration Store connection string
```

Comment:
Could you update the comment up here explain what kind of connection string is expected here?

Reply:
@nicoloboschi done.